geode-core/src/distributedTest/java/org/apache/geode/cache/query/internal/index/InitializeIndexEntryDestroyQueryDUnitTest.java

```diff
@@ -141,7 +141,8 @@ public void testConcurrentRemoveIndexAndQueryOnPR() {
 
   }
 
-  private void waitForAsyncThreadsToComplete(AsyncInvocation asyInvk0, AsyncInvocation asyInvk1) {
+  private void waitForAsyncThreadsToComplete(AsyncInvocation asyInvk0,
```

Comment:
This first `AsyncInvoaction` can also be `AsyncInvocation<Void>`

Reply:
Good catch, missed that one somehow.